lit_gpt/prompts.py

```diff
@@ -61,11 +72,270 @@ def apply(self, prompt: str, **kwargs: str) -> str:
         )
 
 
+class StableLMAlpha(PromptStyle):
+    def apply(self, prompt: str, **kwargs: str) -> str:
```

Comment:
What's an example of using `**kwargs`? Why do we need it?

Reply:
Alpaca has an optional input. See the new test `test_apply_prompts` I added